doc/lucy-cameraman.jl

```diff
@@ -0,0 +1,54 @@
+### lucy-cameraman.jl
+#
+# Copyright (C) 2019 Jakub Wronowski.
+#
+# Maintainer: Mosè Giordano <mose AT gnu DOT org>
+# Keywords: deconvolution, signal processing
+#
+# This file is a part of Deconvolution.jl.
+#
+# License is MIT "Expat".
+#
+### Commentary:
+#
+# This file produces an example image used in Deconvolution.jl documentation.
+#
+### Code:
+
+using Images, TestImages, Deconvolution, Random, FFTW
+
+Random.seed!(42)
+
+img = channelview(testimage("cameraman"))
+
+(N, M) = size(img)
+@assert N == M
+@assert floor(log2(N)) == ceil(log2(N)) # ensure size is power of 2
```

Comment:
There is a function in Base to check this:
```suggestion
@assert ispow2(N) # ensure size is power of 2
```

Reply:
Great, I will update docs before December, I’m traveling now. Maybe someone else will review the algorithm till that moment.